### PR TITLE
minor: added device access FAQ

### PR DESCRIPTION
**Pull request by NatasaKyvetou**
Added `Does balena have access to my device, source code and images?` question

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/faq/troubleshooting/faq.md [hunk 1]
```diff
@@ -143,6 +144,11 @@ There are a few options for devices that do not have an official device type on
...
 
+##### Does {{ $names.company.lower }} have access to my device, source code and images?
+
+Device access is granted to a subset of {{ $names.company.lower }} employees to enable support and device troubleshooting. This access is controlled by ssh key access and only after access is granted to balena by the device fleet manager.
+Application source code and images are stored on {{ $names.company.lower }} backend servers with access limited only to administrative/operational staff and are not exposed to anyone outside of {{ $names.company.lower }}. It is also possible to bypass the {{ $names.company.lower }} builder entirely and push only pre-built artefacts, meaning that {{ $names.company.lower }} never has access to the code at any point.
```
Comment (vipulgupta2048) [inline pages/faq/troubleshooting/faq.md:150]:
```suggestion
Device access is granted to a subset of {{ $names.company.lower }} employees to [enable support and device troubleshooting][support-access]. This access is controlled by ssh key access and only after access is explicitly granted to balena. 
Application source code and images are stored on {{ $names.company.lower }} backend servers with access limited only to administrative/operational staff and are not exposed to anyone outside of {{ $names.company.lower }}. It is also possible to bypass the {{ $names.company.lower }} builder entirely and push only pre-built artifacts, meaning that {{ $names.company.lower }} never has access to the code at any point.
```

Suggested a few changes @NatasaKyvetou 
You can add a link below to the support access document - https://www.balena.io/docs/learn/manage/support-access using the reference style links, you can read more about them here https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md#adding-links-to-the-docs

**Comment by NatasaKyvetou** [comment]
@balena-ci rebase

**Comment by NatasaKyvetou** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Review by georgiats — APPROVED** [review]

**Comment by NatasaKyvetou** [comment]
@balena-ci rebase

**Comment by NatasaKyvetou** [comment]
@balena-ci I self-certify!

**Comment by NatasaKyvetou** [comment]
@vipulgupta2048 I need your help again with this one. It should be ready to be merged and closed, but not sure what went wrong? :D

**Comment by vipulgupta2048** [comment]
@NatasaKyvetou on my side, the PR has already been merged.

**Pull request merged**